tests/unit/api/conftest.py

```diff
@@ -135,11 +147,31 @@ def saved_dimension_table_fixture(snowflake_dimension_table):
     yield snowflake_dimension_table
 
 
+@pytest.fixture(name="snowflake_scd_table")
+def snowflake_scd_table_fixture(
+    snowflake_database_table_scd_table, snowflake_scd_table_id, catalog
+):
+    """SCDTable object fixture"""
+    _ = catalog
+    scd_table = snowflake_database_table_scd_table.create_scd_table(
+        name="sf_scd_table",
+        natural_key_column="col_text",
+        surrogate_key_column="col_int",
+        effective_timestamp_column="effective_timestamp",
+        end_timestamp_column="end_timestamp",
+        current_flag_column="is_active",
+        _id=snowflake_scd_table_id,
+    )
+    assert scd_table.frame.node.parameters.id == scd_table.id
+    yield scd_table
+
+
 @pytest.fixture(name="saved_scd_table")
-def saved_scd_table_fixture(snowflake_scd_table):
+def saved_scd_table_fixture(snowflake_scd_table, catalog):
     """
     Saved SCD table fixture
     """
+    _ = catalog
```

Comment:
having to put the `_ = catalog` everywhere feels kinda annoying, wonder if there's anyway to reduce that boilerplate heh.

Reply:
One possibility is to use autouse property but it's tricky to control the load order of fixtures